test/tests.json

```diff
@@ -1,5 +1,12 @@
 {
   "messages": [
+    {
+      "name": "[enigmail] encrypted iso-2022-jp, plain text",
+      "content": [
+        "ゾし逸現飲"
+      ],
+      "params": "?frameId=none&msgId=16f431a0b9056562&senderEmail=michael.flowcrypt2%40gmail.com&isOutgoing=___cu_false___&acctEmail=flowcrypt.compatibility%40gmail.com&parentTabId=0"
```

Comment:
You can include the message directly using `message=`, so that it's not loaded from the mock. That way it will not rely on mock data.

Reply:
Thanks, I didn't notice that.